src/app/sign-up/sign-up.component.ts

```diff
@@ -1,15 +1,49 @@
 import { Component, OnInit } from '@angular/core';
-
+import {SharedService} from '../shared.service';
+import {CommonService} from '../common.service';
+import {NotificationsService, SimpleNotificationsModule } from 'angular2-notifications';
 @Component({
-  selector: 'app-sign-up',
-  templateUrl: './sign-up.component.html',
-  styleUrls: ['./sign-up.component.css']
+	selector: 'app-sign-up',
+	templateUrl: './sign-up.component.html',
+	styleUrls: ['./sign-up.component.css']
 })
 export class SignUpComponent implements OnInit {
+	user = {firstName:'' ,lastName:'' ,Email:'', mobileNumber:"" ,username:'' ,password:'' ,confirmPassword:''};
```

Comment:
everytime you declare a object, do not declare the keynames just decalare as empty object {}.